careers/careers/utils.py

```diff
@@ -14,10 +14,10 @@ def order_by_function(x):
         order_by = order_by_function
 
     return (
-        sorted(set(
+        sorted(
```

Comment:
Fair enough. This ended up here b/c while testing this PR I got failing tests due to lists containing the same items but with different order. But indeed this has nothing to do with the travis changes. 

I opt to keep this in the same PR to avoid failing tests but I moved it into its separate commit.